cmd/check.go

```diff
@@ -22,6 +22,7 @@ func setupCheckCommand() *cobra.Command {
 			return nil
 		},
 	}
+	cmd.PersistentFlags().BoolP(failFastFlagName, "f", true, "fail if any file requires updates")
```

Comment:
Nit: to reflect the "fast" part of fail fast:

```suggestion
	cmd.PersistentFlags().BoolP(failFastFlagName, "f", true, "fail immediately if any file requires updates")
```

Reply:
Fixed